user-guide/Getting_started/Creating_a_DMS_in_the_cloud.md

```diff
@@ -1,78 +1,73 @@
 ---
-uid: Deploying_a_DMS_in_the_cloud
+uid: Creating_a_DMS_in_the_cloud
 ---
 
-# Deploying a DataMiner System in the cloud (DaaS)
-
-> [!IMPORTANT]
-> At present, this feature is only available in preview for the "Empower" organization.
+# Creating a DataMiner System in the cloud (DaaS)
 
 > [!TIP]
 > If you have any questions or need support with the DaaS feature, contact <daas@dataminer.services>.
 
-With **DataMiner as a Service (DaaS)**, you can deploy a complete DataMiner System (DMS) in mere minutes. The DataMiner System is fully hosted in the cloud, so that you can immediately enjoy all the benefits of being connected to the [dataminer.services platform](xref:AboutCloudPlatform). As DaaS uses our [Storage as a Service (STaaS) solution](xref:STaaS), you will be able to make use of a scalable and user-friendly cloud-native storage platform. Scaling will be easy, both vertically (adding or removing resource), and horizontally (adding or removing nodes), whenever you need.
+With **DataMiner as a Service (DaaS)**, you can create a complete DataMiner System (DMS) in mere minutes. The DataMiner System is fully hosted in the cloud, so that you can immediately enjoy all the benefits of being connected to the [dataminer.services platform](xref:AboutCloudPlatform). As DaaS uses our [Storage as a Service (STaaS) solution](xref:STaaS), you will be able to make use of a scalable and user-friendly cloud-native storage platform. Scaling will be easy, both vertically (adding or removing resource), and horizontally (adding or removing nodes), whenever you need.
 
-## Deploying a new DMS on dataminer.services
+## Creating a new DMS on dataminer.services
+
+> [!IMPORTANT]
+> You must be part of an organization to create a new DaaS instance.
```

Comment:
"part of" sounds a bit odd here. Could I replace this with "a member of"? I'd also add a link to the explanation of what we mean with "organization" to avoid possible confusion.
```suggestion
> You must be a member of an [organization](xref:Pricing_Usage_based_service#organization) to create a new DaaS instance.
```